packages/amazonq/src/lsp/lspInstaller.ts

```diff
@@ -54,3 +55,13 @@ export class AmazonQLspInstaller extends BaseLspInstaller.BaseLspInstaller<
 
     protected override downloadMessageOverride: string | undefined = 'Updating Amazon Q plugin'
 }
+
+export function getBundledResourcePaths(ctx: vscode.ExtensionContext): AmazonQResourcePaths {
+    const assetDirectory = vscode.Uri.joinPath(ctx.extensionUri, 'resources', 'language-server').fsPath
+    return {
+        lsp: path.join(assetDirectory, 'servers', 'aws-lsp-codewhisperer.js'),
+        node: process.execPath,
+        ripGrep: '',
```

Comment:
not sure if this is ok

Reply:
I don't find a way to bundle cross platform ripGrep in VSC as this is anti pattern. VSC expected per platform vsixes. To do that properly, need to split the extension release

Reply:
i think arnab said he was ok with not including ripgrep

Reply:
Updated. We use the rg in the PATH env var, if rg is not found, agentic chat will prompt user to install it